src/gpl/src/nesterovBase.h

```diff
@@ -1123,4 +1133,37 @@ class biNormalParameters
   float uy;
 };
 
+struct GCellIndexHandle
```

Comment:
It would be better to constructors like
```
GCellIndexHandle(NesterovBaseCommon* nbc, size_t index);
GCellIndexHandle(NesterovBase* nbc size_t index);
```
to make the construction clearer.